clang-tools-extra/clangd/ConfigYAML.cpp

```diff
@@ -230,6 +230,10 @@ class Parser {
       if (auto AllScopes = boolValue(N, "AllScopes"))
         F.AllScopes = *AllScopes;
     });
+    Dict.handle("ArgumentLists", [&](Node &N) {
+      if (auto ArgumentLists = scalarValue(N, "ArgumentLists"))
```

Comment:
since `F.ArgumentLists` is itself an `optional`, this can be simplified to:

`F.ArgumentLists = scalarValue(N, "ArgumentLists");`

Reply:
(this comment remains to be addressed)